install-dynamodb-local.sh

```diff
@@ -0,0 +1,23 @@
+
+#!/bin/bash
+
+
+echo "Installing Java"
+brew install openjdk
+
+echo "java is installed sucessfully.... :) "
+echo " "
+echo " "
+
+mkdir ./dynamolocal && cd ./dynamolocal
+
+
+echo "Downloading the DynamoDB tar file" 
+wget http://dynamodb-local.s3-website-us-west-2.amazonaws.com/dynamodb_local_latest.tar.gz
+tar xzf dynamodb_local_latest.tar.gz
+rm -f dynamodb_local_latest.tar.gz
+
+echo " "
+echo "Dynamodb-local is installed and ready for use.... :) "
+      
```

Comment:
Would also be quite cool if we started a Makefile for the project to keep scripts together

If we added:

```
start-dynamodb-local:
	cd dynamolocal && java -Djava.library.path=./DynamoDBLocal_lib -jar DynamoDBLocal.jar -sharedDb
```

To a root file called `Makefile` then we can have the command: `make start-dynamodb-local` that will run this command for us in a shorthand. Not necessary for this PR though, I've also made an issue: 

- https://github.com/openupthecloud/open-cloud-dev-box/issues/31